dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java

```diff
@@ -36,17 +35,16 @@
 @ToString(callSuper = true)
 @EqualsAndHashCode(callSuper = true)
 @JsonDeserialize(builder = TargetRepository.Builder.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class TargetRepository extends TargetRepositoryRef {
```

Comment:
What is the point of now having TargetRepository full DTO?

Reply:
None, it's not even used.

Reply:
And never have been.

Reply:
If we removed it, would it cause issues? It's just for clean code.

Reply:
Removed